server/game/core/card/baseClasses/PlayableOrDeployableCard.ts

```diff
@@ -77,6 +78,19 @@ export class PlayableOrDeployableCard extends Card {
         return true;
     }
 
+    public override canBeInPlay(): this is InPlayCard {
+        return false;
+    }
+
+    public override getSummary(activePlayer: Player, hideWhenFaceup: boolean) {
+        const summary = super.getSummary(activePlayer, hideWhenFaceup);
+        if ((this.canBeInPlay() && this.isInPlay()) || this.location === Location.Resource) {
```

Comment:
Just a note - I think you can simplify this further, it could just be a one liner as:

```typescript
return { ...super.getSummary(activePlayer, hideWhenFaceup), exhausted: this._exhausted };
```

It'll just return `exhausted: null` in cases where exhausted doesn't apply but that's basically the default value that TS gives you when accessing a property that doesn't exist anyway, so it doesn't impact behavior really at all. I guess the difference is that whatever library is sending the data over the wire might still include the `exhausted: null` field in the state which could cause issues if you're not expecting it. Up to you which is better.

Reply:
The issue I ran into with this and damage was that I ran into some contract issues when cards were in hand and inelligable attributes were being accessed.